eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/docbook.iso.sch

```diff
@@ -246,6 +246,7 @@
     <s:rule context="db:link">
       <s:assert test="not(.//db:link)">link must not occur in the descendants of link</s:assert>
       <s:assert test="not(.//db:ezlink)">ezlink must not occur in the descendants of link</s:assert>
+      <s:assert test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))">using scripts in links is not allowed</s:assert>
```

Comment:
question: are the `on*` attributes (onclick, onfocus, ...) kept (not only on links btw) ?